src/frontend/src/components/SearchInput/AuthorSearchInput.jsx

```diff
@@ -1,17 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { FormControl, TextField } from '@material-ui/core';
 
 function AuthorSearchInput({ text, onChange, classes }) {
   return (
     <>
-      <input
-        className={classes.input}
-        list="search-suggestions"
```

Comment:
Removing list when you change it over to a TextField here would effect how the backend support for author autocomplete is handled later on.